source/server/admin/stats_params.h

```diff
@@ -14,12 +14,32 @@
 namespace Envoy {
 namespace Server {
 
+namespace Labels {
```

Comment:
Should the namespace be less generically named (since the enclosing namespaces aren't specifically stat related)?
(I don't feel too strongly about this)

Reply:
Done